.github/workflows/lektor-check.yml

```diff
@@ -0,0 +1,39 @@
+name: lektor check
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
```

Comment:
As the main purpose is whether the website still works, I don't think we need to test that many Python versions. The most recent stable is probably enough. Looking at my system this seems to be the 3.7 branch.

Reply:
Und da wären wir schon bei Problemen.

Du magst altertümliche python versionen wie 3.7 verwenden. Ich verwende 3.8. Travis nimmt 3.6. Andere womöglich noch älter...

Lass uns doch python probleme erkennen in dem wir alle gängigen versionen testen!

Reply:
Wir schreiben hier ja keine Bibliothek für Python, bei der wir ständig testen müssen ob die noch mit allen Versionen kompatibel ist. Prinzipiell würde es Sinn ergeben das mit der Version laufen zu lassen die von den Meisten die die Website bearbeiten verwendet wird, was momentan vermutlich die 3.7 ist, aber meinetwegen können wir auch schon die 3.8 nehmen, der Standard wird sich ja wahrscheinlich eh bald ändern, wenn der Support für 3.7 dieses Jahr abläuft.

Also sagen wir einfach schon `3.8`.

Nachtrag: Ich sehe auch gerade das 3.8 noch garnicht auf der Lektor-Seite als "kompatibel getestet" steht. :man_shrugging:

Reply:
Also eine offiziell nicht supportete pythonversion? Als single source of truth? sicher?